packages/react/src/base-chart.js

```diff
@@ -9,9 +9,4 @@ export default class BaseChart extends React.Component {
 
 		Object.assign(this, this.chart);
 	}
-
-	// TODO
-	// componentWillUnmount() {
-	// 	this.chart.removeChart();
-	// }
```

Comment:
React takes care of this since the chart renders inside a the component container